homeassistant/components/rointe/strings.json

```
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    },
+    "error": {
+      "invalid_auth": "Invalid authentication",
```

This is already defined in `homeassistant/strings.json`.

```suggestion
      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
```